tools/ci_build/build.py

```diff
@@ -870,14 +870,21 @@ def update_submodules(source_dir):
     run_subprocess(["git", "submodule", "update", "--init", "--recursive"], cwd=source_dir)
 
 
-def install_python_deps(numpy_version=""):
-    dep_packages = ["setuptools", "wheel", "pytest"]
-    dep_packages.append(f"numpy=={numpy_version}" if numpy_version else "numpy>=1.16.6")
-    dep_packages.append("sympy>=1.10")
-    dep_packages.append("packaging")
-    dep_packages.append("cerberus")
-    dep_packages.append("psutil")
-    run_subprocess([sys.executable, "-m", "pip", "install", *dep_packages])
+def install_python_deps(requirements_file='requirements.txt',numpy_version=None):
```

Comment:
Or maybe we can totally remove this function.

Reply:
Do we want to keep the NumPy version overwritten capability?  We can remove the function unless we don't want this capability.

Reply:
I think we no longer need to.

Reply:
We could just use the latest numpy